fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -192,6 +192,8 @@ def interface_to_etree(self, port):
                     }
                 }})
 
+        self.apply_trunk_native_vlan_patch(interface_data, port)
```

Comment:
If this exists, the lines 178-179 should not, that code could be in that method call instead of a pass at line 322